jenkins/runGobenchPerformance.fish

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env fish
+set -xg simple (pwd)/performance
+set -xg gobenchdir (pwd)/gobench
+set -xg date (date +%Y%m%d)
+set -xg datetime (date +%Y%m%d%H%M)
+
+if test -z "$ARANGODB_TEST_CONFIG"
+  set -xg ARANGODB_TEST_CONFIG run-small-edges.js
+end
+
+source jenkins/helper.jenkins.fish ; prepareOskar
+
+lockDirectory ; updateOskar ; clearResults
+
+echo Working on branch $ARANGODB_BRANCH of main repository and
+echo on branch $ENTERPRISE_BRANCH of enterprise repository.
+
+switchBranches $ARANGODB_BRANCH $ENTERPRISE_BRANCH true
+if echo "$ARANGODB_BRANCH" | grep -q "^v"
+  pushd work/ArangoDB
+  set -xg date (git log -1 --format=%aI  | tr -d -- '-:T+' | cut -b 1-8)
+  set -xg datetime (git log -1 --format=%aI  | tr -d -- '-:T+' | cut -b 1-12)
+  echo "==== date $datetime ===="
+  popd
+end
+and enterprise
+and maintainerOff
+and releaseMode
+nd buildStaticArangoDB -DTARGET_ARCHITECTURE=nehalem
+
+# clone gobench
+and git clone git@github.com:arangodb/gobench.git
+and cd gobench
+and make
+and cd ..
+
+and rm -rf work/database $simple/results.csv
+and echo "==== starting performance run ===="
+and echo "docker run  -e ARANGO_LICENSE_KEY=$ARANGODB_LICENSE_KEY  -v (pwd)/work/ArangoDB:/ArangoDB  -v (pwd)/work:/data  -v $simple:/performance  -v $gobenchdir:/gobench"
+
+and docker run \
+  -e ARANGO_LICENSE_KEY=$ARANGODB_LICENSE_KEY \
+  -v (pwd)/work/ArangoDB:/ArangoDB \
+  -v (pwd)/work:/data \
+  -v $simple:/performance \
+  -v $gobenchdir:/gobench \
+  arangodb/arangodb \
+  sh -c "cd /performance && \
+    /ArangoDB/build/bin/arangod \
+      -c none \
+      --javascript.app-path /tmp/app \
+      --javascript.startup-directory /ArangoDB/js \
+      --server.rest-server true \
+      --server.endpoint tcp://0.0.0.0:8529 \
+      --javascript.module-directory `pwd` \
+      --server.authentication false \
+      --log.foreground-tty \
+      /data/database & \
+      echo 'Waiting 15s for ArangoDB' && \
+      sleep 15 &&
```

Comment:
typically we loop here with curl requests to _api/version.
As soon as this is available the server runs.

Reply:
from @goedderz 

```

wait_for_arango() {
    local ENDPOINT="$1"

    set +x
    echo "...waiting for curl -s $ENDPOINT/_api/version"

    while ! curl -s "$ENDPOINT"/_api/version
    do
        sleep 0.1
    done

    set -x
}
```
you can hardcode $ENDPOINT here

Reply:
note that this is bash code, I don't know what's necessary to translate it to fish...